### PR TITLE
feat: use relative path

### LINKED ISSUE #5: 自动导入自定义的api，能不能支持根据 resolve.alias 定义路径前缀，而不是本地的文件路径
以仓库中的例子来说就是，
auto-imports.d.ts文件中的
 const bar: typeof import('/Code/Work/vite-auto-import-resolvers/examples/vue/shared/bar')['default']
能不能生成成类似下面这种形式的：
 const bar: typeof import('@/vue/shared/bar')['default'] 或者
 const bar: typeof import('~/vue/shared/bar')['default']
image

### PR DESCRIPTION
**Pull request by jbaubree**
Fixes #5 issue.
Thanks for this plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ export default defineConfig({
         Vue(),
         // 该辅助插件也是必需的 👇
         DirResolverHelper(),
+        dts: 'src/auto-imports.d.ts',
```
Comment (jbaubree) [inline README.md:51]: Need Chinese translate // Set auto-imports.d.ts in your src directory

File: src/dir.ts [hunk 2]
```diff
@@ -20,9 +21,14 @@ interface IGenModulesOptions {
 
 interface Options {
 	/**
-	 * @default 'src'
+	 * Root directory starting from src
```
Comment (jbaubree) [inline src/dir.ts:24]: Need Chinese translate
Root directory starting from src

**Comment by markthree** [comment]
@jbaubree Thanks for your pr，but this seems to complicate the path configuration and limit the location of 'auto-imports.d.ts'

**Comment by markthree** [comment]
@jbaubree Maybe we can use aliases to deal with this 😁

**Comment by markthree** [comment]
For the usage scenario of 'auto-imports.d.ts' flexible location, you can see this project 👉 [tov-template](https://github.com/dishait/tov-template)，But I'm sorry, it's in Chinese 😂

**Comment by jbaubree** [comment]
I will need help for this, i miss something but i cant use alias... Can you complete PR?

**Comment by markthree** [comment]
@jbaubree I opened the [normalize-path](https://github.com/dishait/vite-auto-import-resolvers/blob/main/README_EN.md#normalize-path)，Now you can do more customization in v3.0.0。At the same time, In order to make the configuration easier for users，I removed `root` and only kept `target`。

**Pull request merged**